backend/src/main/java/com/bakdata/conquery/models/query/ColumnDescriptor.java

```diff
@@ -16,13 +18,17 @@
 @Getter
 @Builder
 @ToString
+@NoArgsConstructor
+@AllArgsConstructor
 public class ColumnDescriptor {
 
 	/**
 	 * The name of the column. This label should be generated as a unique label among the columns.
 	 */
 	private String label;
 
+	private String description;
+
```

Comment:
hiermit bilden wir ein Data-Dictionary ab, ggf kann man hier auf vorliegende DataDictionaries zurückgreifen, damit das befüllen nicht per Hand gemacht werden muss

Reply:
Also es würde sich sicher anbieten, das wieder über die Modularisierung zu lösen, aber in conquery kann ich mir das schwer vorstellen.